std_msgs/Header header
duration duration
string arm
